dev.yml

```diff
@@ -8,6 +8,10 @@ type: ruby
 
 up:
   - ruby
+  - custom:
+      name: "Include optional gems with bundle install"
+      met?: 'cat .bundle/config | grep "BUNDLE_WITH: \"optional\""'
+      meet: bundle config set --local with 'optional'
   - bundler
```

Comment:
Is this bundler command still needed?

Reply:
Yes, all the previous `custom` command is doing is adding a config to the bundler, which will be applied when it runs install